Core/src/org/sleuthkit/autopsy/datamodel/FileTypes.java

```diff
@@ -243,4 +249,182 @@ protected void done() {
             }
         }
     }
+
+    /**
+     * Class that is used as a key by NetBeans for creating result nodes. This
+     * is a wrapper around a Content object and is being put in place as an
+     * optimization to avoid the Content.hashCode() implementation which issues
+     * a database query to get the number of children when determining whether 2
+     * Content objects represent the same thing. TODO: This is a temporary
```

Comment:
Please add a reference to JIRA-2823 to the TODO and  add a note about this code to JIRA-2823.